coala_quickstart/info_extractors/PackageJSONInfoExtractor.py

```diff
@@ -0,0 +1,45 @@
+import json
+
+from coala_quickstart.info_extraction.InfoExtractor import InfoExtractor
+from coala_quickstart.info_extraction.Information import (
+    LicenseUsedInfo, ProjectDependencyInfo, IncludePathsInfo, ManPathsInfo,
+    VersionInfo)
+
+
+class PackageJSONInfoExtractor(InfoExtractor):
+    supported_file_globs = ("package.json",)
+    supported_info_kinds = (
+        LicenseUsedInfo,
+        ProjectDependencyInfo,
+        IncludePathsInfo,
+        ManPathsInfo)
+
+    def parse_file(self, fanme, file_content):
+        return json.loads(file_content)
```

Comment:
this can possibly throw exceptions if the input is malformed; can we wrap it around a try-except and return `{}` if there's some error?